### DISCUSSION TITLE
konkrét projekt ötletek, gondolatok

### DISCUSSION THREAD

**Discussion by borazslo** [szentjozsefhackathon/.github#5] [Ideas]
Kipróbáljuk a discussions funkciót. És ha jó dolog, akkor használjuk is majd.

**Comment by vlacko0930** [comment]
Most kerestem, hogy az ötletemet hova írhatnám. Szerintem jó lenne, ha a https://szentjozsefhackathon.github.io nem 404-et dobna, hanem átirányítana a templom oldalára, ahol van elég jó leírás, vagy a GH profil oldalra.
